letsencrypt/client/display/ops.py

```diff
@@ -0,0 +1,140 @@
+"""Contains UI methods for LE user operations."""
+import os
+import sys
+
+import zope.component
+
+from letsencrypt.client import interfaces
+from letsencrypt.client.display import util as display_util
+
+# Define a helper function to avoid verbose code
+util = zope.component.getUtility  # pylint: disable=invalid-name
```

Comment:
Same as https://github.com/letsencrypt/lets-encrypt-preview/pull/250/files#r25067642 (well, almost, here `util` is defined before it is being subsequently used), but still the name is really generic and confisuing